src/middlewared/middlewared/plugins/vm.py

```diff
@@ -1038,12 +1041,16 @@ def random_mac(self):
                         f'{schema_name}.vcpus',
                         'Only one Virtual CPU is allowed in this system.',
                     )
+            elif not flags['intel_vmx'] and not flags['amd_rvi']:
+                raise CallError(
```

Comment:
I think CallError only makes sense for the vm.start here, for create/update it should raise a validation error

Reply:
This only happens during start, they wouldn't be able to create/update as the page wouldn't show up.

Reply:
How? As far as I can see CallError will happen during create/update.